src/CoreMedia/CMTime.cs

```diff
@@ -308,6 +308,16 @@ public static CMTime GetMinimum (CMTime time1, CMTime time2)
 			return CMTimeMinimum (time1, time2);
 		}
 
+		[TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[DllImport (Constants.CoreMediaLibrary)]
+		extern static CMTime CMTimeFoldIntoRange (CMTime time, CMTimeRange foldRange);
+
+		[TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		public static CMTime FoldIntoRange (CMTime time, CMTimeRange foldRange)
```

Comment:
I'm not sure if the `IntoRange` is needed as `Fold` (verb) is already describing the action and swift is using
```
+- Name: CMTimeFoldIntoRange
+  SwiftName: "CMTime.fold(self:intoRange:)"
```